packages/presentation/presentation_publishing/interfaces/fetch/publishes_reload.ts

```diff
@@ -7,10 +7,10 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { PublishingSubject } from '../../publishing_subject';
+import { Observable } from 'rxjs';
 
 export interface PublishesReload {
-  reload$: PublishingSubject<void>;
+  reload$: Observable<void>;
```

Comment:
This was changed so that get_dashboard_api could declare reload$ like below. If its a PublishingSubject then there is a type error because `merge` returns an observable. I could type as `Omit<Observable<void>, 'next'>;` to disallow `next` getting called, which I think is a good idea.

```
reload$: merge(
      unifiedSearchManager.internalApi.controlGroupReload$,
      unifiedSearchManager.internalApi.panelsReload$
    ).pipe(debounceTime(0))
```

Reply:
resolved with 91e0ed97f149d2b66bbf6270b5b1a9e7336d738a